data-format/message-body-aad.md

```diff
@@ -52,7 +52,12 @@ For [non-framed data](#message-body.md#framed-data), the value of this field MUS
 
 ### Content Length
 
```

Comment:
There's actually a third case: final frames.

The reason we went with the wording of "the length of the plaintext data provided to the algorithm is because:

1. For non-framed, it is length of the entire message because it is encrypted in a single call to the encryption algorithm.
1. For standard frames in framed messages, it is the length of the plaintext in the frame, which equal to the message frame length.
1. For final frames in framed messages, it is the length of the plaintext in the frame, which is something greater than or equal to 0 and less than the message frame length.

In each of these cases, the value is the length of the plaintext data provided to the encryption algorithm for which this AAD is constructed.

Reply:
Gotcha, I will update to make each case explicitly clear. Additionally, I found an incorrect statement in message-body.md regarding frame length so I will clean that up and audit the data format files to make sure we are using `Frame Length` consistently. Thanks!

Reply:
Updated. Note that for (3) it is actually "greater than or equal to 0 and less than or equal to the message frame length"